frontend/packages/console-shared/src/components/dashboard/duration-hook.ts

```diff
@@ -1,11 +1,34 @@
 import * as React from 'react';
+import { useTranslation } from 'react-i18next';
 
 export enum Duration {
   ONE_HR = '1 Hour',
   SIX_HR = '6 Hours',
   TWENTY_FOUR_HR = '24 Hours',
 }
 
+export const TranslatedDuration: React.FC<TranslatedDurationProps> = ({ duration }) => {
+  let translatedString;
+  const { t } = useTranslation();
+
+  switch (duration) {
+    case Duration.ONE_HR:
+      translatedString = t('dashboard~1 Hour');
```

Comment:
UXD wants sentence case.
```suggestion
      translatedString = t('dashboard~1 hour');
```